design-docs/pipeline.md

```diff
@@ -124,7 +124,7 @@ mapping:
 Following field actions are supported:
 
 * **move** — move JSON subtree to another node. 
-* **remove** - remove JSON subtree (dst param  is not needed)
+* **remove** - remove JSON subtree (dst param is not needed)
 * **cast** – assign an explicit type to a node (dst param is not needed)
 
 Actions can be chained: 
```

Comment:
we can delete this topic if we didn't have action `cast`. But apply casting if `type` field is specified in field object